# NamedValue.msg - 单个命名数据字段
string name
float64 value
